src/language-html/utils.js

```diff
@@ -14,6 +14,28 @@ const htmlElementAttributes = require("html-element-attributes");
 const HTML_TAGS = arrayToMap(htmlTagNames);
 const HTML_ELEMENT_ATTRIBUTES = mapObject(htmlElementAttributes, arrayToMap);
 
+// https://infra.spec.whatwg.org/#ascii-whitespace
+const HTML_SPACES = new Set(["\t", "\n", "\f", "\r", " "]);
```

Comment:
The spec calls it "whitespace" (always singular, like "code"), not "spaces". So let's call it "whitespace" too. `HTML_WHITESPACE`, `trimLeadingHtmlWhitespace` (`htmlTrimStart` looked good to me too, BTW), etc.

The word "spaces", at least for me, means only this character: " " (e.g. in this sentence: _Whitespace may consist of spaces, tab characters, and line separators._).

And it's "trailing", not "tailing".

Reply:
I'll fix